views/js/runner/plugins/tools/areaMasking/areaMasking.js

```diff
@@ -0,0 +1,157 @@
+    'taoQtiTest/runner/plugins/tools/areaMasking/mask',
+    'tpl!taoQtiTest/runner/plugins/navigation/button'
+], function ($, _, __, hider, pluginFactory, maskComponent, buttonTpl){
+    'use strict';
+
+    /**
+     * Returns the configured plugin
+     */
+    return pluginFactory({
+
+        name : 'area-masking',
+
+        /**
+         * Initialize the plugin (called during runner's init)
+         */
+        init : function init(){
+            var self = this;
+
+            var testRunner = this.getTestRunner();
+            var $container = testRunner.getAreaBroker().getContentArea().parent();
+
+            //keep a ref to all masks
+            this.masks = [];
+
+            //build the control button
+            this.$button = $(buttonTpl({
+                control : 'area-masking',
+                title : __('Covers parts of the item'),
+                icon : 'eye-slash'
+            }));
+
+            //add a new mask each time the button is
+            this.$button.on('click', function (e){
+                e.preventDefault();
+
+                maskComponent()
+                    .on('render', function(){
+
+                        self.masks.push(this);
+
+                        /**
+                         * @event areaMaksing#maskadd
+                         */
+                        self.trigger('maskadd');
+                    })
+                    .on('destroy', function(){
+
+                        self.masks = _.without(self.masks, this);
+
+                        /**
+                         * @event areaMaksing#maskclose
+                         */
+                        self.trigger('maskclose');
+                    })
+                    .init({
+                        x : self.masks.length * 10,
+                        y : self.masks.length * 10
+                    })
+                    .render($container);
+            });
+
+            //start disabled
+            this.disable();
+
+            //update plugin state based on changes
+            testRunner
+                .on('unloaditem', function (){
+                    //remove all masks
+                    _.invoke(self.masks, 'destroy');
+                })
+                .on('enabletools renderitem', function (){
+                    self.enable();
+                })
+                .on('disabletools unloaditem', function (){
+                    self.disable();
+                });
+        },
+
+        /**
+         * Called during the runner's render phase
+         */
+        render : function render(){
+            this.getAreaBroker().getToolboxArea().append(this.$button);
+        },
+
+        /**
+         * Called during the runner's destroy phase
+         */
+        destroy : function destroy(){
+            this.$button.remove();
+        },
+
+        /**
+         * Enable the button
+         */
+        enable : function enable(){
+            this.setState('disabled', false);
```

Comment:
Yes, I've fixed it. But we have a consistency issue with test plugins and component... one use the `disabled` state when the other uses the `enabled` state